flocker/ca/_ca.py

```diff
@@ -272,7 +339,7 @@ def initialize(cls, path, authority):
         # The organizational unit is set to the common name of the
         # authority, which in our case is a byte string identifying
         # the cluster.
-        organizational_unit = authority.certificate.getSubject().CN
+        organizational_unit = authority.credential.certificate.getSubject().CN
```

Comment:
Too many `.`s in this expression.  Perhaps `CertificateAuthority` should have a method that helps out with this part of the certificate generation logic.